loader/CMakeLists.txt

```diff
@@ -437,4 +437,6 @@ endif()
 target_link_libraries(vulkan PRIVATE Vulkan::Headers)
 add_library(Vulkan::Vulkan ALIAS vulkan)
 
-install(TARGETS vulkan)
+install(TARGETS vulkan EXPORT VulkanLoaderConfig)
+set_target_properties(vulkan PROPERTIES EXPORT_NAME "Vulkan")
+install(EXPORT VulkanLoaderConfig DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/VulkanLoader NAMESPACE Vulkan::)
```

Comment:
I picked `Vulkan::Vulkan` due to line 438

I would prefer `Vulkan::Loader`.

> would that mean there is overloading of Vulkan::Vulkan with the Vulkan-Header's?

I don't understand the question.

Reply:
Are you concerned about conflicting with the existing FindVulkan.cmake targets?

Reply:
Hmm my example was going to be any project that uses both Vulkan-Headers and Vulkan-Loader, but I'm not sure there is an issue since the headers uses Vulkan::Headers.

I think there may be an issue for projects that use Vulkan-Loader but find the headers using `find_package(Vulkan REQUIRED)`. However, I'm not sure this is a use-case to worry about. Changing the name of the vulkan target in this repo is possible (changing it to `loader`) but I'm unsure what that might affect as well.

Reply:
I added a second commit I can easily revert that uses Vulkan::Loader instead

Reply:
> Are you concerned about conflicting with the existing FindVulkan.cmake targets?

Yes, that was my concern. Using Vulkan::Loader would be fine then.

Reply:
> I think there may be an issue for projects that use Vulkan-Loader but find the headers using find_package(Vulkan REQUIRED). However, I'm not sure this is a use-case to worry about. 

Yeah I'm not sure if this is a use-case to worry about.

> Changing the name of the vulkan target in this repo is possible (changing it to loader) but I'm unsure what that might affect as well.

I'm not suggesting changing the target. Just the alias name.